### ISSUE TITLE
Uncaught exception on invalid image uploads due to missing sync-method

### ISSUE THREAD

**Issue by aboettger-tuhh** [hedgedoc/hedgedoc#3253]
### Description

A user tries to upload an image and the whole system crashes with an uncaught exception.

### Steps to reproduce

Since I don't know which account wanted to upload the image, I unfortunately don't know which document it is and what kind of file was used. So I can not recreate the problem.

### Expected behaviour

Due to an error at the end user level, the whole system should never be affected.

### Logs

```shell
app_1       | 2023-02-03T09:10:58.732Z info:   46.xxx.xxx.xxx - - [03/Feb/2023:09:10:58 +0000] "POST /uploadimage HTTP/1.1" - - "https://writemd.rz.tuhh.de/Rw7xg7cUTXa7JwwZL1G4WQ?both" "Mo
zilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36"
app_1       | 2023-02-03T09:10:58.732Z error:   Image upload error: formidable error: Error: Request aborted
app_1       | 2023-02-03T09:10:58.737Z error:   uncaughtException: rimraf: callback function required
app_1       | AssertionError [ERR_ASSERTION]: rimraf: callback function required
app_1       |     at rimraf (/hedgedoc/node_modules/rimraf/rimraf.js:56:10)
app_1       |     at /hedgedoc/lib/web/imageRouter/index.js:86:7
app_1       |     at zalgoSafe (/hedgedoc/node_modules/dezalgo/dezalgo.js:20:10)
app_1       |     at f (/hedgedoc/node_modules/once/once.js:25:25)
app_1       |     at IncomingForm.<anonymous> (/hedgedoc/node_modules/formidable/src/Formidable.js:177:9)
app_1       |     at IncomingForm.emit (node:events:513:28)
app_1       |     at IncomingForm._error (/hedgedoc/node_modules/formidable/src/Formidable.js:465:10)
app_1       |     at IncomingMessage.<anonymous> (/hedgedoc/node_modules/formidable/src/Formidable.js:197:14)
app_1       |     at IncomingMessage.emit (node:events:513:28)
app_1       |     at IncomingMessage._destroy (node:_http_incoming:224:10)
app_1       |     at _destroy (node:internal/streams/destroy:102:25)
app_1       |     at IncomingMessage.destroy (node:internal/streams/destroy:64:5)
app_1       |     at abortIncoming (node:_http_server:642:9)
app_1       |     at socketOnClose (node:_http_server:636:3)
app_1       |     at Socket.emit (node:events:525:35)
app_1       |     at TCP.<anonymous> (node:net:301:12)
app_1       | 2023-02-03T09:10:58.738Z error:   An uncaught exception has occured.
app_1       | 2023-02-03T09:10:58.738Z error:   rimraf: callback function required
app_1       | 2023-02-03T09:10:58.738Z error:   Process will exit now.
```

### Your Setup

Docker image: quay.io/hedgedoc/hedgedoc:1.9.6

**Comment by davidmehren** [comment]
Please respect the maintainers efforts by at least trying to file a useful bug report.
This includes a title and a description of what you were doing when the bug happened.

**Comment by aboettger-tuhh** [comment]
> Please respect the maintainers efforts by at least trying to file a useful bug report. This includes a title and a description of what you were doing when the bug happened.

Okay, I got that. I've changed the report.

**Comment by mrdrogdrog** [comment]
Please correct the title as well

**Comment by ErikMichelson** [comment]
Fixed with #3328
